localstack-core/localstack/packages/java.py

```diff
@@ -77,6 +77,8 @@ def _get_install_marker_path(self, install_dir: str) -> str:
         return os.path.join(install_dir, "bin", "java")
 
     def _get_download_url(self) -> str:
+        # Note: Eclipse Temurin does not provide Mac aarch64 Java 8 builds.
+        # See https://adoptium.net/en-GB/supported-platforms/
```

Comment:
Q: This would basically mean that the download will fail because the URL does not exist? If so, could we make the error a bit more transparent?